vendor/angle-bracket-invocation-polyfill/runtime-polyfill.js

```diff
@@ -95,6 +95,10 @@ import { lte, gte } from 'ember-compatibility-helpers';
             }
           }
 
+          runtimeResolver.builtInModifiers = {
```

Comment:
Can you tweak this slightly to:

```js 
runtimeResolver.builtInModifiers = Ember.assign({}, runtimeResolver.builtInModifiers);
```

I believe it should have the same benefit (not mutating the global state), but also preserve any other items in the built in modifiers.

Reply:
Yes, will do.

BTW I've found another leak with `CURLY_COMPONENT_MANAGER` few lines below (https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill/blob/master/vendor/angle-bracket-invocation-polyfill/runtime-polyfill.js#L166-L169), I can update this PR (or create a new one) with something like

```javascript
let customManager = Ember.assign({}, manager);
```

Reply:
I've updated the PR, but left `CURLY_COMPONENT_MANAGER` issue to a separate investigation.

Reply:
Ya, updating CURLY_COMPONENT_MANAGER does make sense to me but I'm happy to just review that in a separate PR.

Reply:
Yeah, I'll try to provide a separate PR, right now I'm still facing with `CURLY_COMPONENT_MANAGER` leak, because using `Ember.assign` breaks setting attributes on components from parent template.